### PR TITLE
browserstack-espresso-app-automate 0.2.0

### PR DESCRIPTION
**Pull request by fernanthonies**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/browserstack-espresso-app-automate/0.2.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,190 @@
+title: browserstack-espresso-app-automate
```
Comment (lpusok) [inline steps/browserstack-espresso-app-automate/0.2.0/step.yml]: Title is an user facing string. Should be in a format similar to `Send Telegram Message`. The displayed strings can be tested by running `bitrise :workflow-editor` in a directory that has a bitrise.yml referencing the step; which is repository root in this case.

File: steps/browserstack-espresso-app-automate/0.2.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  commit: 6ad451d8b6f538b38efdb6308de0fdccd8406d09
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+type_tags:
+- utility
+toolkit:
```
Comment (lpusok) [inline steps/browserstack-espresso-app-automate/0.2.0/step.yml]: This is the default toolkit used, if not specified.

File: steps/browserstack-espresso-app-automate/0.2.0/step.yml (hunk 1 above)
Comment (lpusok) [reply steps/browserstack-espresso-app-automate/0.2.0/step.yml]: Please do a new share/new PR if changing the step.yml, to make sure the commit hash is updated and it references the correct version tag.

Comment (fernanthonies) [reply steps/browserstack-espresso-app-automate/0.2.0/step.yml]: Got it.  I'll clean that up and open a new PR.

**Comment by fernanthonies** [comment]
Closing to address comments.

**Pull request closed without merging**